jwst/outlier_detection/outlier_detection.py

```diff
@@ -81,6 +82,9 @@ def __init__(self, input_models, reffiles=None, **pars):
             partial(Step._make_output_path, None)
         )
 
+        self.hpy = pars['hpy']
```

Comment:
Just curious, what is the "hpy" attribute of pars? I don't see it being used anywhere (but maybe it's just hidden from the diff display).

Reply:
This was originally added to allow the memory inspection object from 'guppy' to be passed around to provide better stats on the memory usage during processing.  All references to this module were removed, except this overlooked reference.  It has been removed in commit 446410ea.